docs/user/lectures.rst

```diff
@@ -0,0 +1,164 @@
+.. _lectures:
+
+Lectures
+========
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+Overview
+--------
+Apart from :ref:`Exercises` and :ref:`Exams`, Artemis can also host the **Lectures** of a course.
+
+Students
+^^^^^^^^
+
+Students can see all published lectures of a course, grouped by date, by clicking on the tab labeled *Lectures*.
+
+|lectures-overview|
+
+Clicking on a lecture leads to following page, where students can see the title, description, lecture units, and attachments.
+
+|lecture-details|
+
+Instructors
+^^^^^^^^^^^
+Instructors can upload lecture slides, divide lectures into units, integrate video streams, lecture recordings, and exercises into lectures, and define learning goals.
+On the course management site, clicking on |lectures-management-btn| opens the following page for managing lectures.
+
+|lectures-management|
+
+Lectures
+--------
+
+Instructors have two options for creating a lecture.
+
+    1. Create a new lecture from scratch by clicking on |create-lecture-btn|.
+    Lectures consist of a title, a description, and optionally a start and end date.
+
+        |create-lecture|
+
+    2. Alternatively, instructors can also import a lecture from a any other course where they at least have *editor* access.
+    Clicking on |import-lecture-btn| opens the import modal, where instructors search for an exisiting lecture and import it.
+
+        |import-lecture|
+
+Once a lecture is created, instructors can add attachments to it.
+An attachment is a file (e.g., document, image) with a title and an optional release date.
+
+|create-attachment|
+
+Lecture Units
+-------------
+
+Lectures can be divided into lecture units, which can be of the following types:
+
+    * **Text unit:** A text with markup.
+    * **Exercise unit:** An exercise from the same course.
+    * **Video unit:** An embedded video stream or video from an external source (e.g., RGB live steam, YouTube video).
```

Comment:
Should we even mention the RBG here? I think it would be better to just mention live streams in general.
```suggestion
    * **Video unit:** An embedded video stream or video from an external source (e.g., RBG live steam, YouTube video).
```